templates/README.md.njk

```diff
@@ -29,6 +29,8 @@ Include an icon using the corresponding CSS class in an empty element:
 
 Add `fl-fw` as class for a fixed width icon.
 
+If you want to insert a glyph of this font on a GNU/Linux system press `Ctrl + Shift + u`, release the keys and then type the code point, for instance: `Ctrl + Shift + u` and `f31a` will insert the `Tux` glyph.
```

Comment:
With Linux I know the shortcuts, but I always have to look them up for WIndows and Mac, maybe we want to add them here also?

Reply:
Changed.